config/crds/verify_v1beta1_metadata.yaml

```diff
@@ -60,6 +62,8 @@ spec:
                   type: string
                 redirectURL:
                   type: string
+                signingCertificate:
```

Comment:
Similarly, perhaps `samlSigningCertificate` would be just slightly clearer. (To help distinguish from metadataSigningCertificate etc.)